src/llm.py

```diff
@@ -332,3 +332,32 @@ async def respond(message: Message) -> None:
                     await message.reply(response)
                 else:
                     await message.channel.send(response)
+
+
+async def generate_album_art(
+    artist: str, title: str, description: Optional[str]
+) -> Optional[str]:
+    prompt = [
+        "Generate bizarre photorealistic album art for the following song.",
+        f"{artist} - {title}\n",
+    ]
+    if description:
+        prompt.append(
+            f"Here is how the artist describes the song: {description[:1000]}"
```

Comment:
```suggestion
            # Cap the description to 1000 characters, to avoid going over any token limits.
            f"Here is how the artist describes the song: {description[:1000]}"
```